### DISCUSSION TITLE
An interface or a video tutorial on how it works?

### DISCUSSION THREAD

**Discussion by MasterLokito** [humhue/DAOTools#12] [Q&A] [answered]
Hello!
I'm hoping to get a video tutorial on how to perform the procedure in other .TLK files to use in other MODS.
My English is basic (Google translate, thank you!) and I didn't really understand how to perform file conversions.
I would also like to know about the possibility of a user-friendly interface to use the tool. I know it's an old game and probably "abandoned", but I still have hope for the interface.

**Comment by humhue** [comment]
Do I get it right that you want to patch some .TLK files? I think that would involve manually translating the lines... so you can do that already with the normal toolset.

But I was working on a technique to possibly let users edit any new line added by mod creators (in any GFF file), I'd just need some beta testers to test if that works because I'm not playing the game anymore.

I was also writing a GUI with all these features but again the main problem is testing stuff. I can't do that. And I don't like releasing broken stuff.

**Reply by MasterLokito** [reply]
Hello!
Thanks for answering.
In fact, I would like to learn how to use the tool to use in other MODS that correct dialogs between others...
Just like you did with QUDAO Fixpack, it can be used in any language of the game.
I use a translation made by community players, the game does not have an official translation into my language.
I remember that with the help of a friend I managed to use his tool in another MOD and it worked perfectly, taking the translation made by the community and correcting some bugs that the MOD proposed, but I forgot how to make the tool work and I no longer have contact with this friend. .
That's why I asked about a video or an interface to use the tool.

I just want to learn how to do this:
"It just removes the reference to the english text."

**Reply by humhue** [reply]
You can try reading this, see if it helps.
I'll eventually finish writing the GUI.

https://github.com/humhue/DAOTools/discussions/6

**Reply by MasterLokito** [reply]
Hi!

Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell to get new features and improvements! https://aka.ms/PSWindows

PS C:\Users\User\DAOMod> pip3 install construct
Requirement already satisfied: construct in c:\users\usuario\appdata\local\programs\python\python312\lib\site-packages (2.10.70)
PS C:\Users\Usuario\DAOMod> git clone https://github.com/humhue/DAOTools.git
git: The term 'git' is not recognized as the name of an operable cmdlet, function, script file, or program. Check the
spelling of the name, or if a path was added, verify that the path is correct and try again.
In line:1 character:1
+ git clone https://github.com/humhue/DAOTools.git
+ ~~~
     + CategoryInfo : ObjectNotFound: (git:String) [], CommandNotFoundException
     + FullyQualifiedErrorId : CommandNotFoundException

I got stuck...

**Reply by humhue** [reply]
You need to install git, alternatively, read this:
The git clone command is needed to download this library, but you can also download it by github (just be sure to unzip the downloaded file if you do so).

**Reply by MasterLokito** [reply]
O Windows PowerShell
Copyright (C) Microsoft Corporation. Todos os direitos reservados.

Instale o PowerShell mais recente para obter novos recursos e aprimoramentos! https://aka.ms/PSWindows

PS C:\Users\Usuario\DAOMod> python DAOTools shale.erf shalenovo.erf
Extracting the files...
Finished extracting in 0.0010285377502441406s!

Patching .cut files...
Finished patching .cut files in 0.05538129806518555s!

Patching .dlg files...
C:\Users\Usuario\AppData\Local\Temp\tmpv6hviwif\den400_playerprison.dlg
Finished patching .dlg files in 0.502983570098877s!

Patching .plo files...
Finished patching .plo files in 0.05186295509338379s!

Building the .erf file...
Finished building in 0.0016987323760986328s!

Total execution time: 0.6143922805786133s
PS C:\Users\Usuario\DAOMod>

Thanks!!!
YahoooO!

**Comment by humhue** [comment, marked as answer]
Thanks to this discussion here, the tool now comes with a GUI. It's downloadable from Nexus.
